tests/e2e/test_main.py

```diff
@@ -336,11 +348,18 @@ def test_ignored_files_are_not_copied(
 
     img_uri_str = f"image:extras-e2e:{uuid.uuid4()}"
 
-    result = cli_runner(
-        ["neuro", "image-build", "-f", str(dockerfile_path), ".", img_uri_str]
-    )
+    try:
+        result = cli_runner(
+            ["neuro", "image-build", "-f", str(dockerfile_path), ".", img_uri_str]
+        )
 
-    assert ignored_file_content not in result.stdout
+        assert ignored_file_content not in result.stdout
+    finally:
+        try:
+            cli_runner(["neuro", "image", "rm", img_uri_str])
+        except Exception as e:
```

Comment:
is it fails frequently? what could be the reason?
(curious, since we do not have this logic in all other image creation tests).

Reply:
Now we have it everywhere :) In general the logic here is that the test might fail before image creation in which case `neuro image rm` fails when it attempts to delete a non-existent image, which we don't care of